# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/spiralsynthmodular/param.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C.org	Mon Jan 20 14:18:44 2003
+++ ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C	Mon Jan 20 14:19:05 2003
@@ -56,7 +56,7 @@
    MaxBox->box (FL_ENGRAVED_BOX);
    MaxBox->color (16);
    add (MaxBox);
-   Meter = new Fl_VU_Meter (2, 62, 226, 20);
+   Meter = new Fl_VU_Meter (2, 62, 226, 20, NULL );
    Meter->color (FL_BLACK);
    cb_Reset_i (Reset, NULL);
    add (Reset);
